src/query/generated/proto/rpcpb/query.proto

```diff
@@ -90,3 +92,45 @@ message M3Segment {
 	int64 startTime = 3;
 	int64 blockSize = 4;
 }
+
+message SearchRequest {
+	bytes target                = 1;
+	SearchType type             = 2;
+	repeated bytes targetTags   = 3;
+	map<string, bytes> metadata = 4;
```

Comment:
remove metadata and just use grpc metadata ?

Reply:
Was considering this, sounds fair going forwards since this is only here to account for some specific internal fields